src/main/java/net/dv8tion/jda/api/entities/MessageSticker.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2015-2020 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.dv8tion.jda.api.entities;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Set;
+
+/**
+ * An object representing a sticker in a Discord message.
+ *
+ * @see Message#getStickers()
+ */
+public class MessageSticker implements ISnowflake
+{
+    private final long id;
+    private final String name;
+    private final String description;
+    private final long packId;
+    private final String asset;
+    private final String previewAsset;
+    private final StickerFormat formatType;
+    private final Set<String> tags;
+
+    private static final String STICKER_URL = "https://cdn.discordapp.com/stickers/%s/%s.png";
+
+    public MessageSticker(final long id, final String name, final String description, final long packId, final String asset, final String previewAsset, final StickerFormat formatType, final Set<String> tags)
+    {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.packId = packId;
+        this.asset = asset;
+        this.previewAsset = previewAsset;
+        this.formatType = formatType;
+        this.tags = tags;
+    }
+
+    @Override
+    public long getIdLong()
```

Comment:
Shouldn't getIdLong also be documented?

Reply:
that's just to override the `ISnowflake` id